README.md

```diff
@@ -67,7 +67,7 @@ Response:
   Status: 201
 ```
 
-#### Mandatory requirement is to publish the application in one of the popular hosting services e.g. Heroku, AWS, etc
+#### Mandatory requirement is to publish the application on AWS. Please use your own account and services with free tiers.
```

Comment:
only AWS? I've been talking to many people regarding this. For most of them it doesn't make a difference where the app is deployed